src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.test.tsx

```diff
@@ -159,4 +160,52 @@ describe('signals/incident/components/IncidentPreview/components/MapPreview', ()
     rerender(withAppContext(<MapPreview {...specificFeatureTypes} />))
     expect(screen.queryByTestId('mapStaticMarker')).not.toBeInTheDocument()
   })
+
+  describe('Location description', () => {
+    it('When an object is selected', () => {
+      render(withAppContext(<MapPreview {...props} />))
+      expect(screen.queryByTestId('locationDescription')).toBeInTheDocument()
+      expect(screen.queryByTestId('typeIcon')).toBeInTheDocument()
+      expect(screen.queryByTestId('selectionLabel')).toBeInTheDocument()
+      expect(screen.queryByTestId('mapAddress')).toBeInTheDocument()
```

Comment:
Please try to make use of other queries over `...ByTestId`. See https://testing-library.com/docs/queries/about#priority.

Reply:
sure